src/css/components/card.scss

```diff
@@ -0,0 +1,27 @@
+@import '../../../node_modules/uswds/src/stylesheets/core/defaults.scss';
+@import '../override_vars.scss';
+@import '../core.scss';
+
+.card {
+  background-color: $color-white;
+  border: 1px solid $color-textblack;
+  border-radius: $rounded;
+  margin-bottom: $grid-2;
+  padding: 0;
+}
+
+.card-home-contents-title {
```

Comment:
These classnames are too deeply nested. Why can't we remove the `home` and `contents` sections, especially considering these cards are only being used in one place.